### ISSUE TITLE
Design Review Calendar - Time Scheduled Pop Up

### ISSUE THREAD

**Issue by RChandler234** [Northeastern-Electric-Racing/FinishLine#2131]
### Description

You should branch off of and merge into this feature branch: `feature/design-review`

Add the Create Design Review Modal. This ticket is blocked by the backend endpoint for design review create and the main design review calendar page being completed, but you can start working on the visuals/ form part of this ticket before they're done. See the mocks for the modal below:

### Acceptance Criteria

- [ ] The modal should allow you to change the required/optional attendees, the date, the time of the design review, and the team
- [x] The time of the design review should be stored as an int (see how we're storing times for design reviews as 15 min increments, ask your tech lead if you're confused)
- [x] The date should save the date with no time/timezone data
- [ ] Team Type is a model that comes from the database, you'll need to call the getAllTeamTypes endpoint to fill out the options for this one (or coordinate with whoever is making the endpoint)

### Proposed Solution

Make sure to use the NERFormModal component we have in FinishLine (might looks a little different than the mock)

### Mocks

![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/cbcf8455-6232-43ed-b466-f0b10edfd814)
